README.md

```diff
@@ -18,8 +18,8 @@ or in detached mode:
 
 You want to contribute to this project? Great!
 
-[Drafts & Documentation](https://github.com/E-Edu/draft-documents)<br>
-[Git Workflow (German)](https://github.com/E-Edu/general/blob/master/guides/conventions.md)<br>
+[Drafts & Documentation](https://github.com/E-Edu/draft-documents) <br>
+[Git Workflow (German)](https://github.com/E-Edu/general/blob/master/guides/conventions.md) <br>
```

Comment:
You should change this also and request the creation of it.

Reply:
I will change it if its moved